src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -47,7 +47,7 @@ public class SFSession extends SFBaseSession {
       "CLIENT_STORE_TEMPORARY_CREDENTIAL";
   private static final ObjectMapper mapper = ObjectMapperFactory.getObjectMapper();
   private static final int MAX_SESSION_PARAMETERS = 1000;
-  private static final int DEFAULT_HTTP_CLIENT_SOCKET_TIMEOUT = 300000; // millisec
+  public static final int DEFAULT_HTTP_CLIENT_SOCKET_TIMEOUT = 300000; // millisec
```

Comment:
Nit, I know this already exists but can this be renamed to `DEFAULT_HTTP_CLIENT_SOCKET_TIMEOUT_MS`?

Reply:
@sfc-gh-hchaturvedi @sfc-gh-mknister do you guys ok with more style changes?